app/javascript/packs/admin/members/memberControls.js

```diff
@@ -0,0 +1,68 @@
+import { INTERACTIVE_ELEMENTS_QUERY } from '@utilities/dropdownUtils';
+
+const expandSearchButton = document.getElementById('expand-search-btn');
+const expandFilterButton = document.getElementById('expand-filter-btn');
+const searchSection = document.getElementById('search-users');
+const filterSection = document.getElementById('filter-users');
+
+expandSearchButton?.addEventListener('click', () => {
+  collapseControlsSection({
+    section: filterSection,
+    triggerButton: expandFilterButton,
+  });
+
+  expandOrCollapseControlsSection({
+    section: searchSection,
+    triggerButton: expandSearchButton,
+  });
+});
+
+expandFilterButton?.addEventListener('click', () => {
+  collapseControlsSection({
+    section: searchSection,
+    triggerButton: expandSearchButton,
+  });
+
+  expandOrCollapseControlsSection({
+    section: filterSection,
+    triggerButton: expandFilterButton,
+  });
+});
+
+/**
+ * Ensures the given controls section is closed.
+ *
+ * @param {HTMLElement} section The controls section to be closed
+ * @param {HTMLElement} triggerButton The button responsible for opening and closing the section
+ */
+const collapseControlsSection = ({ section, triggerButton }) => {
+  section?.classList.add('hidden');
+  triggerButton?.setAttribute('aria-expanded', false);
+};
+
+/**
+ * Expands the given section if it's currently closed. Otherwise closes it.
+ *
+ * @param {HTMLElement} section The controls section to be toggled
+ * @param {HTMLElement} triggerButton The button responsible for opening and closing the section
+ */
+const expandOrCollapseControlsSection = ({ section, triggerButton }) => {
+  const isExpanded = triggerButton?.getAttribute('aria-expanded') === 'true';
+  if (isExpanded) {
+    section?.classList.add('hidden');
+    triggerButton?.setAttribute('aria-expanded', false);
+  } else {
+    section?.classList.remove('hidden');
+    triggerButton?.setAttribute('aria-expanded', true);
+    sendFocusToFirstInteractiveItem(section);
+  }
```

Comment:
to avoid if/else we can maybe instead: 

```suggestion
  if (isExpanded) {
    section?.classList.add('hidden');
    triggerButton?.setAttribute('aria-expanded', false);
    return;
  }

section?.classList.remove('hidden');
triggerButton?.setAttribute('aria-expanded', true);
sendFocusToFirstInteractiveItem(section);
```

Reply:
Thank you for the suggestion! My personal view is that a basic `if/else` statement is more readable when there's a simple `true/false` condition like this with similar/related operations (e.g. in this case "show it" or "hide it"). Although I know an early return might look nicer to some eyes.

I know folks might have differing opinions on early returns, so if anyone feels strongly about this, just let me know and I can change 🙂